From 52151e23889cea0d41f44b9b35d9216786992ff1 Mon Sep 17 00:00:00 2001
From: Tim Walker <tdskywalker@gmail.com>
Date: Sat, 11 Jan 2014 23:08:23 +0100
Subject: [PATCH] vc1: enable the interlaced B-frame codepath.

This should have been done in 1fb013a57c6d98d3f425506eb25f32b2cf7ddc6f, but was missed because FFmpeg had moved that code elsewhere.

Signed-off-by: Tim Walker <tdskywalker@gmail.com>
---
 libavcodec/vc1.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/libavcodec/vc1.c b/libavcodec/vc1.c
index 319230a..acefce6 100644
--- a/libavcodec/vc1.c
+++ b/libavcodec/vc1.c
@@ -1188,7 +1188,6 @@ int ff_vc1_parse_frame_header_adv(VC1Context *v, GetBitContext* gb)
             if (v->bfraction == 0) {
                 return -1;
             }
-            return -1; // This codepath is still incomplete thus it is disabled
         }
         if (v->extended_mv)
             v->mvrange = get_unary(gb, 0, 3);
-- 
1.8.5.1

